README.md

```diff
@@ -1,17 +1,68 @@
----
-description: 'The unofficial Discord.js beginner''s guide, written by idiots for beginners.'
----
+<h1 align='center'>An Idiots Guide</h1>
+<h4 align='center'>Discord.JS Bot Guide</h4>
+<p align='center'>
+    <em>The unofficial Discord.JS beginner's guide, written by idiots, for beginners</em>
+</p>
```

Comment:
Couldn't you just wrap this with a `div` tag with the `align` prop?

Reply:
I don't think MD would recognize a div tag... Worth a shot though..

Reply:
I have now learned you can use a div tag in MD lol

Thank you!!